src/feature/wines/components/wine-progress/index.tsx

```
@@ -1,56 +1,39 @@
 
 const WineProgress = ({ wineId }: WineProgressProps) => {
-  console.log(wineId);
-  // TODO: API 연동 시 windId를 받아 커스텀 훅으로 데이터 가져와 사용
-  const getPercentage = (count: number) =>
-    mockData.reviewCount === 0 ? 0 : (count / mockData.reviewCount) * 100;
-  const avgRating = getAverageRating(mockData);
+  const { data: wineDetail } = useWineDetail({ wineId, enabled: !!wineId });
+  const reviewCount = wineDetail?.reviewCount || 0;
+  const avgRatings = wineDetail?.avgRatings || {};
+  const getPercentage = (count: number = 0) =>
+    reviewCount === 0 ? 0 : (count / reviewCount) * 100;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)